rubicon/client/dataframe.py

```diff
@@ -60,14 +67,14 @@ def plot(self, **kwargs):
         >>> dataframe.plot(kind='line', x='Year', y='Number of Subscriptions')
         """
         try:
-            # data is a dask dataframe
             import hvplot.dask  # noqa F401
+            import hvplot.pandas  # noqa F401
```

Comment:
can we only try the import thats related to the given `kind`? if I say `kind='pandas'` and it says "you don't have dask" I'd be pretty confused